app/client/src/components/pages/Community.Tabs.Protect.js

```diff
@@ -292,7 +296,7 @@ function Protect() {
   // Updates the visible layers. This function also takes into account whether
   // or not the underlying webservices failed.
   const updateVisibleLayers = useCallback(
-    ({ key = null, newValue = null, useCurrentValue = false }) => {
+    ({ newValues = [], useCurrentValue = false }) => {
```

Comment:
The `useCurrentValue` parameter can be removed, as it looks like it isn't being used.

Reply:
Hmm that was originally there, and it's used on line 374, so I'll test if it's needed. But the `newValues` doesn't need to be an array, I can switch that back.

Reply:
Oops, I missed that one. Sorry about the confusion.

Don't worry about testing it, everything is good as is. I'll approve and merge once this other build completes.